server/src/main/scala/za/co/absa/atum/web/api/controller/BaseApiController.scala

```diff
@@ -25,9 +26,14 @@ import za.co.absa.atum.web.api.service.BaseApiService
 import za.co.absa.atum.web.model.BaseApiModel
 
 import java.net.URI
-import java.util.concurrent.CompletableFuture
 import java.util.{Optional, UUID}
-import javax.servlet.http.HttpServletRequest
+import java.util.concurrent.CompletableFuture
+//import javax.servlet.http.HttpServletRequest
+
+//import java.net.URI
+//import java.util.concurrent.CompletableFuture
+//import java.util.{Optional, UUID}
+//import javax.servlet.http.HttpServletRequest
```

Comment:
since all this is actually imported above, all these can be removed I believe?

Reply:
But all this where added to enhance the implicit call usage and return types. So, they are not really the same.

Reply:
import jakarta.servlet.http.HttpServletRequest
import java.net.URI
import java.util.{Optional, UUID}
import java.util.concurrent.CompletableFuture
//import javax.servlet.http.HttpServletRequest

//import java.net.URI
//import java.util.concurrent.CompletableFuture
//import java.util.{Optional, UUID}
//import javax.servlet.http.HttpServletRequest